client/my-sites/plugins/plugins-browser-list/test/index.jsx

```diff
@@ -25,29 +20,38 @@ const plugins = [
 const props = {
 	plugins,
 	variant: PluginsBrowserListVariant.Fixed,
-	listName: 'woocommerce',
-	title: 'woocommerce',
-	subtitle: '100 plugins',
+	listName: 'arbitrary-plugin-list-name',
+	title: 'arbitrary-plugin-list-title',
+	subtitle: 'arbitrary-plugin-list-subtitle',
 	site: {
 		plan: PLAN_FREE,
 	},
 	size: 6,
 };
 
+const queryPlaceholders = () => {
+	const listItems = screen.queryAllByRole( 'listitem' );
```

Comment:
One downside of using this helper function is that we're using `query*` queries for elements that exist, which is [not recommended by the library](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence), but I'm happy to take this compromise in favor of reusability 👍